modules/installation-initializing-manual.adoc

```diff
@@ -134,12 +134,21 @@ it in the `<installation_directory>`.
 You must name this configuration file `install-config.yaml`.
 ====
 ifdef::restricted[]
+
 ** Unless you use a registry that {op-system} trusts by default, such as
 `docker.io`, you must provide the contents of the certificate for your mirror
 repository in the `additionalTrustBundle` section. In most cases, you must
 provide the certificate for your mirror.
 ** You must include the `imageContentSources` section from the output of the command to
 mirror the repository.
+** The 'ImageContentSourcePolicy' is generated as an output of `oc-mirror` flow (after the mirroring process is finished).
```

Comment:
```suggestion
** The 'ImageContentSourcePolicy' is generated as an output of `oc mirror` flow after the mirroring process is finished.
```

Reply:
So, this is interesting. The downloadable module is called "oc-mirror". It provides the command "oc mirror".